x8.md

```diff
@@ -1,34 +1,38 @@
 ---
 sip: 3
-title: Initial Message Payload Specification Specification
+title: Initial Message Payload Specification
 status: Draft
 type: Standard
-author:
+author: Adam Babik <adam@status.im>, Oskar Thorén <oskar@status.im>
 created: 2019-04-22
 updated:
 ---
 
-**TODO: Remove everyting that doesn't touch on payload here**
-
-Using Adam's spec as starting point. Also be mindful of integration with the overview document.
-
-**TODO: Remove non transport privacy layer aspects here**
-
+- [Payload](#payload)
+- [Encoding](#encoding)
+  - [Future encoding format](#future-encoding-format)
+- [Content types](#content-types)
+- [Message types](#message-types)
+- [Clock vs Timestamp and message ordering](#clock-vs-timestamp-and-message-ordering)
+- [Replies](#replies)
+- [Upgradability](#upgradability)
 
-# Payload
+This specification describes how the payload of each message in the Status Protocol looks like. It does not care how the payload is encrypted or exchanged later.
```

Comment:
Briefly on purpose, like to enable chat communication, things like stickers etc and uniform through many clients

Pointer to whitepaper here maybe? Enable SNT use cases outside of more conventional chat use cases